WooCommerce/WooCommerceUITests/Screens/Login/LoginEpilogueScreen.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let displayNameField = "full-name-label"
+    static let siteUrlField = "url-label"
+    static let continueButton = "login-epilogue-continue-button"
+}
+
+class LoginEpilogueScreen: BaseScreen {
```

Comment:
It's good practice to mark classes as `final` to be explicit that this class should never be inherited.

```suggestion
final class LoginEpilogueScreen: BaseScreen {
```